www/src/public/js/controllers/builder/main.js

```diff
@@ -1184,6 +1374,25 @@
 
                 return;
             }
+
+            //if the new item is not a runecharm, set that charm slot to default AAAAA
+            if($scope.selectedList.items[$scope.currentItemIndex].id === -5 && angular.copy(item).id != -5) {
```

Comment:
I added this because if you change a runecharm to a standard item, the charm string for that slot will remain and get saved/loaded into the lsitstring. This way the string gets reset to default when a different item is selected.